.github/workflows/package_installers.yml

```diff
@@ -133,6 +136,19 @@ jobs:
            --app-image "${{ env.MACOS_TARGET_PATH }}" \
            --dest ${{ env.MACOS_TARGET_DEST_PATH }}
 
+      - name: Package cli app jar with Gradle
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: shadowJar
+
+      - name: Persist cli app jar
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: "gtfs-validator-cli-${{ env.SHORT_SHA }}"
```

Comment:
[suggestion]: add `zip` extension to let the users know what they are downloading.

Reply:
✅ done